diff --git a/node_modules/auth0-extensions-cli/lib/client-config.js b/node_modules/auth0-extensions-cli/lib/client-config.js
index 47c8987..e526168 100644
--- a/node_modules/auth0-extensions-cli/lib/client-config.js
+++ b/node_modules/auth0-extensions-cli/lib/client-config.js
@@ -1,7 +1,7 @@
 const path = require('path');
 const webpack = require('webpack');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
 const StatsWriterPlugin = require('webpack-stats-plugin').StatsWriterPlugin;

 const productionMode = 'production';
@@ -140,8 +140,8 @@ module.exports = ({ name, version }, rootPath, mode, entry, destination) => {
   switch (mode) {
     case productionMode:
       config.optimization.minimizer = [
-        new UglifyJsPlugin({
-          uglifyOptions: {
+        new TerserPlugin({
+          terserOptions: {
             mangle: true,
             output: {
               comments: false
diff --git a/node_modules/auth0-extensions-cli/lib/config.js b/node_modules/auth0-extensions-cli/lib/config.js
index fed5db1..b65a04f 100644
--- a/node_modules/auth0-extensions-cli/lib/config.js
+++ b/node_modules/auth0-extensions-cli/lib/config.js
@@ -1,7 +1,7 @@
 const _ = require('lodash');
 const path = require('path');
 const semver = require('semver');
-const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
 const Webpack = require('webpack');

 function convertDependenciesToCommonJS(dependencies) {
@@ -103,14 +103,11 @@ module.exports = (pkg, rootPath, args, ext) => {
   }

   activePlugins.push(
-    new UglifyJSPlugin({
-      uglifyOptions: {
+    new TerserPlugin({
+      terserOptions: {
         ecma,
-        compress: {
-          warnings: false,
-          unused: true,
-          dead_code: true
-        },
+        compress: false,
+        mangle: false,
         output: {
           comments: false
         }
